okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java

```diff
@@ -44,6 +47,8 @@
   private final ModuleStore moduleStore;
   private Vertx vertx;
   private final Messages messages = Messages.getInstance();
+  // tenants with new permission module (_tenantPermissions version 1.1 or later)
+  private ConcurrentMap<String, Boolean> newPermModuleTenants = new ConcurrentHashMap<>();
```

Comment:
Can `private Set<String> expandedPermModuleTenants = new ConcurrentHashMap<String, Boolean>().newKeySet()` been used instead to avoid the unused Boolean?

Reply:
Done.